tailwind/tailwind.css

```
@@ -602,32 +602,55 @@
   }
 
-  .base-table tbody tr:first-child::after {
+  .base-table tbody tr::after {
     content: '';
-    @apply -translate-1/2 absolute left-1/2 top-[53.8%] -z-10 h-[80%] w-[99%] -translate-y-1/2 rounded-lg bg-gray-100 opacity-0;
-  }
-
-  .base-table tbody tr:hover::after {
-    @apply opacity-100;
+    @apply -translate-1/2 absolute left-1/2 -z-10 w-[99%] rounded-lg bg-gray-100 opacity-0;
+    top: var(--row-hl-top);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)